src/React.Core/ReactComponent.cs

```diff
@@ -94,12 +99,25 @@ public virtual string RenderHtml(bool renderContainerOnly = false, bool renderSe
 						: string.Format("ReactDOMServer.renderToString({0})", GetComponentInitialiser());
 					html = _environment.Execute<string>(reactRenderCommand);
 				}
-				return string.Format(
-					"<{2} id=\"{0}\">{1}</{2}>",
-					ContainerId,
-					html,
-					ContainerTag
-				);
+				if (string.IsNullOrEmpty(ContainerClass))
+				{
+					return string.Format(
+						"<{2} id=\"{0}\">{1}</{2}>",
+						ContainerId,
+						html,
+						ContainerTag
+						);
+				}
+				else
+				{
+					return string.Format(
+						"<{2} id=\"{0}\" class=\"{3}\">{1}</{2}>",
```

Comment:
Rather than having two separate branches here, it might be worth having a list of attributes

Something like (not tested):
```csharp
var attributes = id=\"" + ContainerId + "\";
if (!string.IsNullOrEmpty(ContainerClass))) {
	attributes += " class=\"" + ContainerClass + "\"";
}

return string.Format(
	"<{2} {0}>{1}</{2}>",
	attributes,
	html,
	ContainerTag
);
```